latest 5.2.0 fails to build with Qt 6.6.0

Hi,

I tried on Mageia Cauldron to build latest kvirc 5.2.0 but it fails to build with the following error:

`/home/iurt/rpmbuild/BUILD/KVIrc-5.2.0/src/kvilib/ext/KviRegExp.cpp: In member function 'QString KviRegExp::getCompletePattern() const':
/home/iurt/rpmbuild/BUILD/KVIrc-5.2.0/src/kvilib/ext/KviRegExp.cpp:37:134: error: invalid conversion from 'int' to 'QRegularExpression::WildcardConversionOption' [-fpermissive]
   37 |                 return QRegularExpression::wildcardToRegularExpression(m_szPattern, QRegularExpression::UnanchoredWildcardConversion | QRegularExpression::NonPathWildcardConversion);`

This is with Qt 6.6.0

Attached the full build log:
[build.x86_64.0.20240120155705.log](https://github.com/kvirc/KVIrc/files/13998456/build.x86_64.0.20240120155705.log)

--- a/src/kvilib/ext/KviRegExp.cpp
+++ b/src/kvilib/ext/KviRegExp.cpp
@@ -33,7 +33,7 @@ QString KviRegExp::getCompletePattern() const
 {
 	if(m_ePs == PatternSyntax::Wildcard)
 	{
-#if QT_VERSION >= QT_VERSION_CHECK(6, 6, 0)
+#if QT_VERSION >= QT_VERSION_CHECK(6, 6, 1)
 		return QRegularExpression::wildcardToRegularExpression(m_szPattern, QRegularExpression::UnanchoredWildcardConversion | QRegularExpression::NonPathWildcardConversion);
 #elif QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
 		QString tmp = QRegularExpression::wildcardToRegularExpression(m_szPattern, QRegularExpression::UnanchoredWildcardConversion);

